pytest_mypy_plugins/utils.py

```diff
@@ -368,6 +369,10 @@ def extract_output_matchers_from_out(out: str, params: Mapping[str, Any], regex:
     return matchers
 
 
+def render_template(template: str, data: Mapping[str, Any]) -> str:
+    return chevron.render(template=template, data={k: v if v is not None else "None" for k, v in data.items()})
```

Comment:
Chevron renders `None` as "" rather than as "None", which is probably the right thing for a general purpose 'tash engine, but not what we need.

Reply:
Do we need a test for that?

Reply:
It's already covered by `test-parametrized.yml`, or I wouldn't have found out that it was needed!

Reply:
I'm happy to write a pure unit test if you'd like one, though.

Reply:
No, this is fine! Thanks! 👍

Reply:
Too late - I wrote one. :-)